docs/admin/kubeadm.md

```diff
@@ -387,6 +453,42 @@ started.
 Once the cluster is up, you can grab the admin credentials from the master node
 at `/etc/kubernetes/admin.conf` and use that to talk to the cluster.
 
+## Cloudprovider integrations (experimental)
+
+Enabling specific cloud providers is a common request. This currently requires
+manual configuration and is therefore not yet fully supported. If you wish to do
+so, edit the kubeadm dropin for the kubelet service
+(`/etc/systemd/system/kubelet.service.d/10-kubeadm.conf`) on all nodes,
+including the master. If your cloud provider requires any extra packages
+installed on the host, for example for volume mounting/unmounting, install those
+packages.
```

Comment:
Should also note that there is other configuration that may be needed (IAM roles for AWS) that is currently underdocumented.